domains/qwertx.is-an.app.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "../schemas/domain.schema.json",
+  "description": "My personal site and other stuff",
+  "domain": "is-an.app",
+  "subdomain": "qwertx",
+  "owner": {
+    "repo": "https://github.com/notqwertz/free-domains",
```

Comment:
Is that repo with your site content?

```suggestion
    "repo": "https://github.com/NotQWertz/NotQWertz.github.io",
```

Reply:
Yeah

Reply:
Update this property in this case, please :)